### ISSUE TITLE
Application of type VRApplication_Background not listed in controller bindings ui

### ISSUE THREAD

**Issue by peroht** [ValveSoftware/openvr#1337]
On SteamVR 1.10.28 I can no longer see applications of type EVRApplicationType::VRApplication_Background in the controller bindings configuration UI.

I've tested both

"SteamVR Settings > Controllers > Show Old Binding UI"

and 

"SteamVR Settings > Controllers > Manage Controller Bindings".

Does the appliaction have to have some additional meta data tied to ie? For example, the background application in question which used to be visible in the old UI used to have no icon in the list. It was however selectable and configurable.

**Comment by BOLL7708** [comment]
Aw man, you did it, you got issue #1337! Congratulations! 🏆

I've made a number of background applications, and I've not had issues editing bindings for them lately, though I'm on the SteamVR beta. Are you loading an app manifest? I had to do that to get input working at all though 🤔

You do mention that even the old interface will not list the application, which was my fallback before it started working, this as the new one doesn't appear to include everything. 👀💦 I'll have to double check this later.
